plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/PinotQueryBuilder.java

```diff
@@ -103,19 +103,18 @@ private static void generateFilterPql(StringBuilder pqlBuilder, PinotTableHandle
 
     public static Optional<String> getFilterClause(TupleDomain<ColumnHandle> tupleDomain, Optional<String> timePredicate, boolean forHavingClause)
     {
+        checkState(!tupleDomain.isNone(), "Pinot does not support 1 = 0 syntax, as a workaround use <column> != <column>");
```

Comment:
Sure: one of the sides of a filter must contain an expression with a column in it, constants on both sides are not supported (throws a "BAD REQUEST" error), so instead of `1 = 0` you can do `<column> != <column>`. lmk if you need any more info.